bc_obps/registration/decorators.py

```diff
@@ -34,3 +36,22 @@ def wrapper(request: HttpRequest, *args: Any, **kwargs: Any) -> Any:
         return wrapper
 
     return decorator
+
+
+def try_except_wrapper():
```

Comment:
This should probably reflect what it's doing in the name! Maybe something like `with_http_errors()`?